librz/util/str.c

```diff
@@ -986,6 +986,27 @@ RZ_API size_t rz_str_ncpy(char *dst, const char *src, size_t dst_size) {
 #endif
 }
 
+/**
+ * \brief Secure string concat with null terminator
+ *
+ * 	This API behaves like strlcat.
+ */
+RZ_API size_t rz_str_ncat(char *dst, const char *src, size_t dst_size) {
```

Comment:
i'm confused why this was added if it already exists?
https://github.com/rizinorg/rizin/blob/dev/librz/util/str.c#L973